docker/scylla-sct/ubuntu/Dockerfile

```diff
@@ -44,3 +37,8 @@ stderr_logfile=/dev/stderr\n\
 stderr_logfile_maxbytes=0' > /etc/supervisord.conf.d/scylla-manager.conf && \
     echo "autostart=false" >> /etc/supervisord.conf.d/scylla-server.conf && \
     echo "autostart=false" >> /etc/supervisord.conf.d/scylla-housekeeping.conf
+
+RUN source /etc/os-release && \
+    grep "PRETTY_NAME=" /etc/os-release | grep -q "Ubuntu 22.04" && \
```

Comment:
won't it better to check the sshd version ? like we do in:
https://github.com/scylladb/scylla-cluster-tests/blob/504d9c0f15eb7a005dc48e131f0b34ec7ea66ba9/sdcm/provision/common/utils.py#L126

Reply:
or something like this:
```bash
SSH_VERSION=$(ssh -V 2>&1 | cut -f 1  --delimiter=' ' | tr -d "[:alpha:][:blank:]_")
dpkg --compare-versions "$SSH_VERSION" ">=" "8.8"
```